<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewUser</class>
 <widget class="QDialog" name="AddNewUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>606</width>
        <height>1022</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="commonInformation">
         <property name="title">
          <string>Загальна інформація</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="userName">
              <property name="placeholderText">
               <string>Ім'я</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="userSecondName">
              <property name="placeholderText">
               <string>Призвіще</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="userMidelName">
              <property name="placeholderText">
               <string>Побатькові</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Дата народження</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="bornDate">
              <property name="displayFormat">
               <string>dd.MM.yyyy</string>
              </property>
              <property name="date">
               <date>
                <year>1998</year>
                <month>1</month>
                <day>1</day>
               </date>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Сатать</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="genderBox">
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <item>
               <property name="text">
                <string>Жіноча</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Чоловіча</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="universitiInformation">
         <property name="title">
          <string>Академічна інформація</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="gradebook">
              <property name="placeholderText">
               <string>Номер залікової книжки</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="academicGroup">
              <property name="placeholderText">
               <string>Академічна група</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Форма навчання: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="isContract">
              <item>
               <property name="text">
                <string>Бюджетна</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Контрактна</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="torismInformation">
         <property name="title">
          <string>Інформація віндносно секції &quot;Туризм&quot;</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QGroupBox" name="ContactInformation">
            <property name="title">
             <string>Контактна інформація</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Адресса проживання: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="address"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Прописка:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="registration">
                 <item>
                  <property name="text">
                   <string>Харківська</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Харківська,  гуртожиок</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Харківська область</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Інше місто</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLineEdit" name="phoneNumbers">
                 <property name="placeholderText">
                  <string>Контактні номери</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="email">
                 <property name="placeholderText">
                  <string>Електронна пошта</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLineEdit" name="groupT">
              <property name="placeholderText">
               <string>Группа на секції туризм</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Спортивний розряд</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="sportCategory">
              <item>
               <property name="text">
                <string>Відсутній</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Мастер спорут</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Кандидат в майтсри спорту</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Перший розряж</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Другий розряд</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Третій розряд</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Додаткова інформація</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QPlainTextEdit" name="additionalInfor">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>500</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddNewUser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddNewUser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
